### DISCUSSION TITLE
I am new here

### DISCUSSION THREAD

**Discussion by krishna1jk** [community/community#55397] [New to GitHub]
Good Morning,
My name is krishna and I am new here . I am trying to use some codes here but I am not a programmer. I am looking for some help in running the files here. If anyone want to help me ,in opening the folders on an IDE and running the codes as is , will be compensated. My interests are vedic Astrology and AI.

Thanks,
Krishna.V
[Redacted due to PII]

**Comment by queenofcorgis** [comment]
Hi @krishna1jk , we're happy you're here 😀

However, we currently do not allow job listings on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
